dapp/src/components/AccountListener.js

```diff
@@ -10,6 +10,9 @@ import { useStoreState } from 'pullstate'
 import { setupContracts } from 'utils/contracts'
 import { login } from 'utils/account'
 
+const BALANCE_CHECK_INTERVAL = 30000 // ms
```

Comment:
Do we think users are going to stare at their balance that much?
If yes, as opposed to pulling that frequently from the contract, would it be worth to "simulating" the balance growing by artificially increasing it by a tiny amount on the DApp without calling the contract? CC @micahalcorn for product input

Reply:
Perhaps move this commentary to the issue #240?